open_instruct/rejection_sampling/rejection_sampling.py

```diff
@@ -50,6 +51,8 @@ class Args:
     hf_entity: Optional[str] = None
     hf_repo_id: str = "rejection_sampling"
     add_timestamp: bool = True
+    mode: str = "judgement"
+    skill: str = "chat"
```

Comment:
Still not used

Reply:
It's being used in `api_generate.py`, it reads those two through `args`